pcdsdevices/sqr1.py

```diff
@@ -1,8 +1,23 @@
 """
-    Ophyd class for square-one tri-sphere motion system
-    (https://www.sqr-1.com/tsp.html)
+Ophyd class for square-one tri-sphere motion system (https://www.sqr-1.com/tsp.html) # noqa: E501
```

Comment:
For the record you shouldn't just ignore line length conventions because you feel like it.  It's perfectly possible to break this up into two lines